packages/orchestration/test/facade-durability.test.ts

```diff
@@ -1,15 +1,15 @@
 import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
+import { makeIssuerKit } from '@agoric/ertp';
+import { reincarnate } from '@agoric/swingset-liveslots/tools/setup-vat-data.js';
```

Comment:
Have we considered adopting eslint `sort-imports`? Feel like I've seen this type of change frequently in recent reviews

Reply:
We have :) https://github.com/Agoric/agoric-sdk/issues/7403

Feel free to lobby to prioritize it.